docs/documentation/colors.md

```diff
@@ -1,3 +1,7 @@
 # Colors
 
-{{colorschemes}}
\ No newline at end of file
+## Named colors
+Named colors in Makie.jl (e.g., `:blue`) are parsed using [Colors.jl](https://juliagraphics.github.io/Colors.jl/stable/constructionandconversion/#Color-Parsing) and thus have a large array of possibilities under CSS specifications. You can find a plotted table of all possible names in this page: https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#color_keywords 
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#color_keywords

that should also be a link I think

Reply:
```suggestion
Named colors in Makie.jl (e.g., `:blue`) are parsed using [Colors.jl](https://juliagraphics.github.io/Colors.jl/stable/constructionandconversion/#Color-Parsing) and thus have a large array of possibilities under CSS specifications. You can find a plotted table of all possible names [in this page](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#color_keywords).
```